docs/source/AdministratorGuide/InstallingAndConfiguringBasics/index.rst

```diff
@@ -0,0 +1,51 @@
+.. _installing-configuring-basics:
+
+==========================================
+Installing and configuring: basic concepts
+==========================================
+
+As seen in :ref:`dirac-setup-structure`, DIRAC provides you with several *components*,
+these components are organized in *systems*, and these components can be installed in a :ref:`server_installation`
+using the :ref:`system-admin-console`.
+
+The components don't need to be all resident on the same host, in fact it's common practice to have several hosts
+for large installations.
+
+As a general rul, services can be duplicated, 
+meaning you can have the same service running on multiple hosts, thus reducing the load.
+There are only 2 cases of DIRAC services that have a "master/slave" concept, and these are the Configuration Service 
+and the Accounting/DataStore service.
+
+Same can be said for executors: you can have many residing on different hosts.
+
+The same can't be said for agents. Some of them can be duplicated, BUT require a proper configuration, 
+and for this you need to read further in the guide.
```

Comment:
You may want to link the scalability page ?

Reply:
Which one?

Reply:
http://dirac.readthedocs.io/en/latest/AdministratorGuide/scalingAndLimitations.html